test/services/notices/setup/submit-select-recipients.service.test.js

```diff
@@ -21,13 +21,17 @@ const SubmitSelectRecipientsService = require('../../../../app/services/notices/
 describe('Notices - Setup - Submit Select Recipients Service', () => {
   let payload
   let recipients
+  let referenceCode
   let session
   let sessionData
   let yarStub
 
   beforeEach(async () => {
     payload = { recipients: ['123'] }
-    sessionData = {}
+
+    referenceCode = `RINV-${Math.floor(1000 + Math.random() * 9000).toString()}`
```

Comment:
Guess this can use the new `generateReferenceCode()`? 😁

Reply:
I don't even think this is necessary here. the data is in a session object so it will never conflict.